ParseXML.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Xml;
+
+namespace TheP0ngServer
+{
+    public class ParseXML
+    {
+        public static int Port;
+        public static string IP;
```

Comment:
It’s better off using encapsulation data hiding. Implement Getters and Setter and use automatic property to modify it later